util/libcrypto.num

```diff
@@ -5440,3 +5440,15 @@ BIO_ADDR_dup                            ?	3_1_0	EXIST::FUNCTION:SOCK
 CMS_final_digest                        ?	3_1_0	EXIST::FUNCTION:CMS
 OPENSSL_strcasecmp                      ?	3_0_3	EXIST::FUNCTION:
 OPENSSL_strncasecmp                     ?	3_0_3	EXIST::FUNCTION:
+hpke_kg                                 ?	3_0_0	EXIST::FUNCTION:
+hpke_kg_evp                             ?	3_0_0	EXIST::FUNCTION:
+hpke_enc                                ?	3_0_0	EXIST::FUNCTION:
+hpke_enc_evp                            ?	3_0_0	EXIST::FUNCTION:
+hpke_dec                                ?	3_0_0	EXIST::FUNCTION:
+hpke_expansion                          ?	3_0_0	EXIST::FUNCTION:
+hpke_ah_decode                          ?	3_0_0	EXIST::FUNCTION:
+hpke_suite_check                        ?	3_0_0	EXIST::FUNCTION:
+hpke_good4grease                        ?	3_0_0	EXIST::FUNCTION:
+hpke_str2suite                          ?	3_0_0	EXIST::FUNCTION:
+hpke_prbuf2evp                          ?	3_0_0	EXIST::FUNCTION:
+hpke_setlibctx                          ?	3_0_0	EXIST::FUNCTION:
```

Comment:
Public functions should have OSSL_HPKE prefix instead of just hpke

Reply:
Looking at this now so that e.g. ``hpke_kg()`` would become ``OSSL_HPKE_kg()`` but also have a question: is there a similar need for #defines and typedefs? E.g. I currently have a #define for ``HPKE_MODE_PSK`` and a ``hpke_suite_t`` type - should I rename those (and similar) too and if so according to what style?

Reply:
thanks, will follow that (there might be >1 commit before I claim cleanliness, but I'll put a note somewhere here when I figure one's clean)

Reply:
think I fixed the naming stuff in https://github.com/openssl/openssl/pull/17172/commits/71df83a99fa7793a62b638b0fd2db7a247cd06d4 - there might be a few more coding-style things to fix yet though